src/config.c

```diff
@@ -37,7 +37,8 @@ static const struct config_default defaults[] = {
     { CFG_VIEWER,       CFG_VIEW_TRANSP,    "grid"                   },
     { CFG_VIEWER,       CFG_VIEW_SCALE,     "optimal"                },
     { CFG_VIEWER,       CFG_VIEW_POSITION,  "center"                 },
-    { CFG_VIEWER,       CFG_VIEW_AA,        "mks13"                  },
+    { CFG_VIEWER,       CFG_VIEW_AA_ON,     "mks13"                  },
+    { CFG_VIEWER,       CFG_VIEW_AA_START,  "yes"                    },
```

Comment:
Now we have inconsistent settings: I can set aliasing to `none` and enable it. Looks slightly confusing.